### PR TITLE
KVM release 15.1.0

### PR DESCRIPTION
**Pull request by tfussell**
Release to address https://github.com/giantswarm/giantswarm/issues/19003

### CODE REVIEW

File: kvm/v15.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+This release includes changes in calico and etcd that should lead to better performance.
+
+## Change details
+
+
+### calico [3.20.1](https://github.com/projectcalico/calico/releases/tag/v3.20.1)
+
+#### Bug fixes
+ - Updated ubi base images and CentOS repos to stop CVE false positives from being reported. [node #1214](https://github.com/projectcalico/node/pull/1214) (@mgleung)
```
[the remark this answers is not in the document]
Comment (tfussell) [reply kvm/v15.1.0/README.md]: Done.

File: kvm/v15.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+ - Improve error message for conflicting routes in CNI plugin [cni-plugin #1164](https://github.com/projectcalico/cni-plugin/pull/1164) (@mgleung)
+ - Bugfix: blackhole routing table with No-OIF / InterfaceNone-only is clobbering all other routes in the same routing table because if netlink.RT_FILTER_OIF is specified with a netlink.Route{LinkIndex: 0}, it will return all routes using the remaining applicable filter (netlink.RT_FILTER_TABLE / Table 254) link routes. [felix #2995](https://github.com/projectcalico/felix/pull/2995) (@electricjesus)
+ - Fix slow performance when updating a Kubernetes namespace when there are many Pods (and in turn, slow startup performance when there are many namespaces). [felix #2967](https://github.com/projectcalico/felix/pull/2967) (@fasaxc)
+ - Fixes a benign error caused by attempting to delete the same IPAMBlock twice. [kube-controllers #822](https://github.com/projectcalico/kube-controllers/pull/822) (@caseydavenport)
+
+
+
+### etcd [3.5.0](https://github.com/etcd-io/etcd/releases/tag/v3.5.0)
+
+Not found
```
[the remark this answers is not in the document]
Comment (tfussell) [reply kvm/v15.1.0/README.md]: Done.

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**